### ISSUE TITLE
Upload multiple files using REST API, Also form-data can't pass the other fields of media collection, it need raw body

### ISSUE THREAD

**Issue by gor3a** [payloadcms/payload#9737]
### Describe the Bug

I tried to upload multiple files using REST API that mentioned in docs [Upload Files](https://payloadcms.com/docs/upload/overview#uploading-files), the single file works well 

![Image](https://github.com/user-attachments/assets/ee0e82a0-5421-4e12-b33b-5954458a8f8b)

But I got error 
```bash
[16:29:04] ERROR: Cannot read properties of undefined (reading 'includes')
    err: {
      "type": "TypeError",
      "message": "Cannot read properties of undefined (reading 'includes')",
      "stack":
          TypeError: Cannot read properties of undefined (reading 'includes')
              at generateFileData (webpack-internal:///(rsc)/./node_modules/.pnpm/payload@3.2.2_graphql@16.8.1_monaco-editor@0.52.0_react-dom@19.0.0-rc-65a56d0e-20241020_react_qv6xnetjprece3gvrppmkpo4cq/node_modules/payload/dist/uploads/generateFileData.js:163:27)
              at createOperation (webpack-internal:///(rsc)/./node_modules/.pnpm/payload@3.2.2_graphql@16.8.1_monaco-editor@0.52.0_react-dom@19.0.0-rc-65a56d0e-20241020_react_qv6xnetjprece3gvrppmkpo4cq/node_modules/payload/dist/collections/operations/create.js:80:145)
              at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
              at async Object.create (webpack-internal:///(rsc)/./node_modules/.pnpm/@payloadcms+next@3.2.2_graphql@16.8.1_monaco-editor@0.52.0_next@15.0.0_react-dom@19.0.0-rc-65_thuhuskbxwhezen25znrjkqmke/node_modules/@payloadcms/next/dist/routes/rest/collections/create.js:29:15)
              at async eval (webpack-internal:///(rsc)/./node_modules/.pnpm/@payloadcms+next@3.2.2_graphql@16.8.1_monaco-editor@0.52.0_next@15.0.0_react-dom@19.0.0-rc-65_thuhuskbxwhezen25znrjkqmke/node_modules/@payloadcms/next/dist/routes/rest/index.js:487:19)
```

### Link to the code that reproduces this issue

pnpx create-payload-app@latest -t blank

### Reproduction Steps

1. Create a blank project with media collection
2. Enable bulkUpload in upload config
3. using /api/media endpoint and passing `file` param with multiple file using postman or any alternative with request type `form-data`

### Which area(s) are affected? (Select all that apply)

area: core

### Environment Info

```text
Payload: 3.2.2
Next: 15.0.0
Nodejs: 20.18.1
```

**Comment by hackerart** [comment]
Same issue even when I'm trying to use x-www-urlencoded or form-data with single file using postman. Need help.![Image](https://github.com/user-attachments/assets/ec85c9b3-ae71-4b9f-a59f-d73fb89d6715)

**Comment by Fady-cmd** [comment]
> Same issue even when I'm trying to use x-www-urlencoded or form-data with single file using postman. Need help. ![Image](https://github.com/user-attachments/assets/8de0d74d-ebd0-4483-b05e-6551f8a2ea48) ![Image](https://github.com/user-attachments/assets/ec85c9b3-ae71-4b9f-a59f-d73fb89d6715)

You need to pass the data as raw; I faced it also, @hackerart

**Comment by hackerart** [comment]
> > Same issue even when I'm trying to use x-www-urlencoded or form-data with single file using postman. Need help. ![Image](https://github.com/user-attachments/assets/8de0d74d-ebd0-4483-b05e-6551f8a2ea48) ![Image](https://github.com/user-attachments/assets/ec85c9b3-ae71-4b9f-a59f-d73fb89d6715)
> 
> You need to pass the data as raw; I faced it also, @hackerart 

But what if I need to use form-data to upload files? I can't upload it using raw.

**Comment by gor3a** [comment]
> > > Same issue even when I'm trying to use x-www-urlencoded or form-data with single file using postman. Need help. ![Image](https://github.com/user-attachments/assets/8de0d74d-ebd0-4483-b05e-6551f8a2ea48) ![Image](https://private-user-images.githubusercontent.com/29780283/393328933-ec85c9b3-ae71-4b9f-a59f-d73fb89d6715.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM2NzMzMTksIm5iZiI6MTczMzY3MzAxOSwicGF0aCI6Ii8yOTc4MDI4My8zOTMzMjg5MzMtZWM4NWM5YjMtYWU3MS00YjlmLWE1OWYtZDczZmI4OWQ2NzE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA4VDE1NTAxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI2ZmVjZmIwMTJlYjRhMmRiZmVkZTQyYTU3YmQwYmI2YzIxNWViMzgxZDZjYjMxNDBjNWFkMzNjMjlmNDA5NTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.w18PgwNkyahX7jUE1fXAPM3nwjOU5aL5gXOvok7kbdw)
> > 
> > 
> > You need to pass the data as raw; I faced it also, [@hackerart](https://github.com/hackerart)
> 
> But what if I need to use form-data to upload files? I can't upload it using raw.

Yeah you are correct, I think is a bug or issue, I think if the media collection has another fields like `deviceId` we need to append two fields in form-data `file` and `deviceId`, @hackerart

**Comment by gor3a** [comment]
@jmikrut Can you check this issue?

**Comment by JessRynkar** [comment]
This issue hasn’t had activity in a while and relates to several releases ago, so we’re going to close it for now. If it’s still occurring on the latest version, please open a new issue with updated reproduction steps and environment details so we can investigate.
